index.js

```diff
@@ -1,97 +1,135 @@
-                        published_date: publishedDate || 'Unknown'
-                    },
-                    description: description || 'Unknown',
-                    img: img || 'Unknown'
-                };
-            };
-            createBlock(books);
-        },
-        type: 'GET'
+    return new Promise((resolve, reject) => {
+        let xhr = new XMLHttpRequest();
+        xhr.open('GET', src);
+        xhr.responseType = 'json';
+        xhr.addEventListener('load', () => {
+            resolve(xhr.response);
+        });
+        xhr.send();
     });
 };
+
+document.getElementById('btnSerch').addEventListener('click', (e) => {
+    serch = document.getElementById('bookName').value;
+    if (document.querySelector('div.panel-success')) {
+        clean();
+    };
+    getBooksInit();
+    navBar.style.display = 'block';
+});
+document.getElementById('clean').addEventListener('click', (e) => {
+    clean();
+    startIndex = 0;
+});
+
 function modalContent(src) {
     document.querySelector('.modal-header').innerHTML = `<button class='close'
     data-dismiss='modal'>x</button>
     <h4 class='text-center text-primary'>${src.title}</h4>`;
-    let book = '';
+    let bookSummary = '';
     for (let key in src) {
-        book += `<span class='capitalize'>${key}</span>:  ${src[key]};<br>`;
+        bookSummary += `<span class='capitalize'>${key}</span>:  ${src[key]};<br>`;
     };
-    document.querySelector('#bodyModal').innerHTML = book;
+    document.querySelector('#bodyModal').innerHTML = bookSummary;
 };
 document.querySelector('#bookshelf').addEventListener('click', (e) => {
     if (e.target.getAttribute('class') === 'btn btn-info openModal') {
-        let id = e.target.id;
-        modalContent(books[id].summary);
+        let idItem = responseResult.find((item) => {
+            return item.id === e.target.getAttribute('data-id');
+        });
+        modalContent(idItem.summary);
         $('#modal').modal();
-    };
+    }
 }, false);
-document.getElementById('btnSerch').addEventListener('click', (e) => {
-    serch = document.getElementById('bookName').value;
-    if (document.querySelector('div.panel-success')) {
+
+document.querySelector('.pagination').addEventListener('click', (e) => {
+    if (e.target.getAttribute('data-id') === 'more') {
+        startIndex += STEP;
+        if (moreCount < PAGE_MAX) {
+            getBooksInit();
+            moreCount++;
+        } else {
+            document.querySelector('button.disabled').setAttribute('class', 'btn btn-default');
+            let parent = document.querySelector('#bookshelf');
+            responseResult.splice(0, STEP);
+            for (let i = 0; i < STEP; i++) {
+                parent.removeChild(parent.firstChild);
+            };
+            getBooksInit();
+            moreCount++;
+        };
+    } else if (e.target.getAttribute('data-id') === 'next') {
+        if (document.querySelector('button.disabled')) {
```

Comment:
`document.querySelector('button.disabled')` - repeated code and can be reusable, create variable for this btn in the beginning of the method.